### ISSUE TITLE
Rename to "contributions today"

### ISSUE THREAD

**Issue by xanderdunn** [tadeuzagallo/GithubPulse#47]
GithubPulse showed that I had made 1 commit today, but I was pretty sure I hadn't made any commits.  Indeed, I looked online and saw that I had simply opened an issue on a project.  These also count as contributions that keep the streak going.  This should be renamed to "contributions today" rather than "commits today".

**Comment by tadeuzagallo** [comment]
Yep, I tried that, but sadly it doesn't fit and it looks super bad with two lines :disappointed: 

I'll try to think of a better way of displaying that.

**Comment by rafaell-lycan** [comment]
@tadeuzagallo I'll try to figure out how do that without make it ugly.
